src/vma/event/event_handler_manager.cpp

```diff
@@ -372,7 +372,10 @@ void event_handler_manager::stop_thread()
 {
 	if (!m_b_continue_running)
 		return;
+
+	m_reg_action_q_lock.lock();
 	m_b_continue_running = false;
+	m_reg_action_q_lock.unlock();
```

Comment:
@serglelv it is not clear what should be fixed using this.

Reply:
Helgrind reports possible data race during write.